templates/python/.github/workflows/python-poetry-release.yaml

```diff
@@ -0,0 +1,27 @@
+name: Default reusable workflow
+
+on:
+  push:
+    branches: [main]
+
+jobs:
+  call-workflow-passing-data:
+    uses: bakdata/ci-templates/.github/workflows/python-poetry-release.yaml@main
+    with:
+      release-type: patch # (Required) See more values at: https://python-poetry.org/docs/cli/#version
+      ref: my-awesome-ref # (Optional) if not set the ${{ github.event.repository.default_branch }} will fill the value. In this case the changes will be pushed to my-awesome-ref
```

Comment:
I think you can remove this because you're setting a branch in the _on push_ trigger

Reply:
done